package.json

```diff
@@ -75,7 +75,6 @@
     "redux": "^3.6.0",
     "redux-mock-store": "^1.3.0",
     "redux-thunk": "^2.2.0",
-    "reselect": "*",
```

Comment:
Was getting a duplication warning on install for these since they're already specified as dependencies.

Reply:
I'm thinking these should be peer dependencies. 
We can probably remove the lodash dependency all together. It's only doing an "isFunction" check and that can be replaced - https://stackoverflow.com/a/7356528

Reply:
Doesn't that mean we'll be requiring the consumers of `redux-waiter` to have `reselect` as a dependency in _their_ project? I felt like that requirement wasn't mandatory for the consumer since `redux-waiter` only needs it for it's own inner workings.

Reply:
right right... 👍